scene/gui/graph_node.cpp

```diff
@@ -29,6 +29,7 @@
 /*************************************************************************/
 
 #include "graph_node.h"
+#include "editor_scale.h"
```

Comment:
Keep in mind that `scene/` stuff is used in games... So you can't use editor stuff here. And you should avoid breaking compat if possible.

Reply:
Fixed !